### ISSUE TITLE
Count doesn't work on built-in functions in mappings

### ISSUE THREAD

**Issue by ipod825** [Jelmerro/Vieb#54]
It doesn't work when I try to map
```vim
pmap d <10pointer.scrollDown>
pmap u 10<pointer.scrollUp>
```

Instead I need to do it directly:
```vim
pmap DD <pointer.scrollDown>
pmap UU <pointer.scrollUp>
pmap d 10DD
pmap u 10UU
```

**Comment by Jelmerro** [comment]
I can already see why this is happening, thanks for finding this. I will start working on this starting today.

**Comment by Jelmerro** [comment]
This was fixed in 8f35d45b402cfbfe001070feb4c3949080971daf
